doc/classes/InputEventMouseMotion.xml

```diff
@@ -16,7 +16,8 @@
 			Represents the pressure the user puts on the pen. Ranges from [code]0.0[/code] to [code]1.0[/code].
 		</member>
 		<member name="relative" type="Vector2" setter="set_relative" getter="get_relative" default="Vector2( 0, 0 )">
-			The mouse position relative to the previous position (position at the last frame).
+			The mouse position relative to the previous position (position at the last frame). 
```

Comment:
Tiny nitpick, there is one trailing space at the end of the line. This is why it appears as modified:

```suggestion
			The mouse position relative to the previous position (position at the last frame).
```

Reply:
@Calinou Shall I remove the space and commit again? Or leave it as it is?

Reply:
Yes, please amend your commit with the space removed if you can.

Reply:
I've removed the extra space.